packages/model/src/dsl.ts

```diff
@@ -105,17 +106,34 @@ export function Index (kind: IndexKind) {
 export function Model<T extends Obj> (
   _class: Ref<Class<T>>,
   _extends: Ref<Class<Obj>>,
-  domain?: Domain
+  domain?: Domain,
+  _implements?: Ref<Interface<Obj>>[]
 ) {
   return function classDecorator<C extends new () => T> (constructor: C): void {
     const txes = getTxes(constructor.prototype)
     txes._id = _class
     txes.extends = _class !== core.class.Obj ? _extends : undefined
+    txes.implements = _implements
     txes.domain = domain
     txes.kind = ClassifierKind.CLASS
   }
 }
 
+/**
+ * @public
+ */
+export function ModelInterface<T extends Obj> (
```

Comment:
Let's change `ModelInterface` to `Implements`? for better model look

Reply:
Done.